nautobot/docs/release-notes/version-1.6.md

```diff
@@ -8,6 +8,25 @@ This document describes all new features and changes in Nautobot 1.6.
 
 ### Added
 
+#### Installation Metrics
+
+A new setting, [`INSTALLATION_METRICS_ENABLED`](../configuration/optional-settings.md#installation_metrics_enabled), has been added to allow Nautobot to send anonymous installation metrics to the Nautobot maintainers. This setting is `True` by default but can be changed in `nautobot_config.py` or the `NAUTOBOT_INSTALLATION_METRICS_ENABLED` environment variable.
+
+If the [`INSTALLATION_METRICS_ENABLED`](../configuration/optional-settings.md#installation_metrics_enabled) setting is `True`, running the [`post_upgrade`](../administration/nautobot-server.md#post_upgrade) or [`send_installation_metrics`](../administration/nautobot-server.md#send_installation_metrics) management commands will send a list of all installed [plugins](../plugins/index.md) and their versions, as well as the currently installed Nautobot version to the Nautobot maintainers. A randomized uuid will be generated and saved in the [`DEPLOYMENT_ID`](../configuration/optional-settings.md#deployment_id) setting to anonymously and uniquely identify each installation. The plugin names will be one-way hashed with SHA256 to further anonymize the data sent.
```

Comment:
Feels like there should be a better way to explain what we're doing with plugin names, i.e. making it possible for us to recognize public/well-known plugins, but keeping the name of any private/proprietary plugins unknown?